lib/sqlalchemy/dialects/oracle/base.py

```diff
@@ -7,8 +7,7 @@
 # mypy: ignore-errors
 
 
-r"""
-.. dialect:: oracle
+r""".. dialect:: oracle
```

Comment:
not 100% sure if sphinx likes it (if not it was updated in the other files too)

Reply:
I had a feeling this was done by the formatter, but maybe I fat-fingered it.

Reply:
maybe it's not an issue, I haven't tried to render the docs. but maybe better safe than sorry.